external/netstandard20/netstandard20.depproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard2.0;
```

Comment:
Because the project is *named* nestandard 2.0.  I presume create a new directory/files to handle a new netstandard version.